ansible/dualtor/y_cable_simulator_client.j2

```diff
@@ -103,26 +103,26 @@ def _get_side(physical_port):
     Args:
         physical_port: physical port on switch, an integer starting from 1
     Returns:
-        1 if TOR_A is active
-        2 if TOR_B is active
+        1 if UPPER_TOR is active
```

Comment:
How about we return strings `UPPER_TOR`/`LOWER_TOR` instead of 1/2. This makes the caller more readable perhaps.

Reply:
I think it was originally designed this was to be compatible with the actual cable driver. @bingwang-ms can you confirm?

Reply:
Yes. We can't change the return value because we want to act as the real driver. Thanks